src/pages/resultsView/coExpression/CoExpressionTab.tsx

```
@@ -138,7 +122,13 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
 
     @bind
     private onSelectGeneticEntity(geneticEntityId:string) {
-        this._selectedGeneticEntityId = geneticEntityId;
+        if (this.props.store.geneticEntities.isComplete) {
```

you might consider throwing an error here because the UI should never allow a user to selected a geneticEntity before geneticEntites are complete.  but not critical

---

@alisman Done

---

its better to throw an error object so that the stack trace is available which i believe its not if you just throw a string

---

its better to throw an error object so that the stack trace is available which i believe its not if you just throw a string